tamr_client/schema_mapping/_schema_mapping.py

```diff
@@ -0,0 +1,31 @@
+"""
+Tamr - Schema Mapping
+See https://docs.tamr.com/new/docs/overall-workflow-schema
+
+The terminology used here is consistent with Tamr UI terminology
+
+Asynchronous versions of each function can be found with the suffix `_async` and may be of
+interest to power users
+"""
+from tamr_client import operation
+from tamr_client._types import Operation, SchemaMappingProject, Session
+from tamr_client.dataset import unified
+
+
+def update_unified_dataset(
+    session: Session, project: SchemaMappingProject
+) -> Operation:
+    """Apply changes to the unified dataset and wait for the operation to complete
+
+    Args:
+        project: Tamr Schema Mapping project
+    """
+    op = _update_unified_dataset_async(session, project)
+    return operation.wait(session, op)
+
+
+def _update_unified_dataset_async(
```

Comment:
Same comment about inlining this helper into `update_unified_dataset`. Less code to maintain 😄

Reply:
Adding the same change in `_mastering` to this PR now.